### ISSUE TITLE
some problem in my windows pc

### ISSUE THREAD

**Issue by DualWield** [vuejs/vue-cli#1]
```
$ npm install -g vue-cli
npm WARN deprecated metalsmith-templates@0.7.0: This package was split into two
simpler plugins: metalsmith-layouts and metalsmith-in-place
C:\Users\DualWield\AppData\Roaming\npm\vue -> C:\Users\DualWield\AppData\Roaming\npm\node_modules\vue-cli\bin\vue
vue-cli@1.0.1 C:\Users\DualWield\AppData\Roaming\npm\node_modules\vue-cli
├── uid@0.0.2
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── chalk@1.1.1 (escape-string-regexp@1.0.4, supports-color@2.0.0, ansi-styles@2.1.0, has-ansi@2.0.0, strip-ansi@3.0.0)
├── read-metadata@1.0.0 (yaml-js@0.0.8)
├── rimraf@2.5.0 (glob@6.0.2)
├── request@2.67.0 (aws-sign2@0.6.0, forever-agent@0.6.1, tunnel-agent@0.4.2, oauth-sign@0.8.0, is-typedarray@1.0.0, caseless@0.11.0, stringstream@0.0.5, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.2.1, qs@5.2.0, node-uuid@1.4.7, combined-stream@1.0.5, mime-types@2.1.8, form-data@1.0.0-rc3, bl@1.0.0, hawk@3.1.2, http-signature@1.1.0, har-validator@2.0.3)
├── download-github-repo@0.1.3 (download@0.1.19)
└── khaos@0.9.3 (is-utf8@0.2.1, thunkify@2.1.2, extend@1.3.0, cp@0.1.1, deep-extend@0.2.11, is@3.1.0, async@0.9.2, metalsmith-build-date@0.1.0, unyield@0.0.1, sort-object@1.0.0, chalk@0.4.0, fs-extra@0.8.1, mkdirp@0.4.2, co-fs-extra@1.1.0, prompt-for@1.1.2, metalsmith-prompt@0.0.8, moment@2.10.6, download-github-repo@0.0.1, metalsmith@1.7.0, to-case@1.0.1, metalsmith-templates@0.7.0, handlebars@2.0.0, lodash@3.10.1, gnode@0.1.1)
```

and then whatever commands I input, get a error like this

```
$ vue init webpack my-project


events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn c:\Users\DualWield\AppData\Roaming\npm\node_modules\vue-cli\bin\vue-init ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
    at Function.Module.runMain (module.js:469:11)
    at startup (node.js:134:18)
    at node.js:961:3


$ vue list

events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn c:\Users\DualWield\AppData\Roaming\npm\node_modules\vue-cli\bin\vue-list ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
    at Function.Module.runMain (module.js:469:11)
    at startup (node.js:134:18)
    at node.js:961:3
```

My env environment is 
```
$ node --version
v4.2.1

$ npm --version
2.14.7
```

win7 + git bash

Thanks

**Comment by yyx990803** [comment]
Possibly due to executable permission issues. Try installing 1.0.2.

**Comment by natocTo** [comment]
Same issue.. I had same also with khaos installer

windows10 + cmd :)

**Comment by JoshZA** [comment]
Same issue here, Evan

win 8.1 x64
node v4.2.1
npm 2.14.7

**Comment by etune** [comment]
Same issue here as well (Windows 10). 
```
node 5.3.0
npm 3.3.12
vue 1.0.2
```

**Comment by yyx990803** [comment]
Should be fixed now in 1.0.3.
